index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+  integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+  crossorigin="anonymous" referrerpolicy="no-referrer" />
+ <link rel="stylesheet" href="style.css">
+ <title>My prortfolio</title>
+</head>
+
+<body>
+ <header>
+  <div class="navbar">
+   <h1 class="logo">My logo</h1>
+   <div class="navbar"><i class="fa-solid fa-bars"></i></div>
+  </div>
+ </header>
+ <section class="main-section">
+  <h1>I’m Naji
+   Glad to see you!</h1>
+  <p class="description">
+   I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+   experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
```

Comment:
Please, like the previous reviewer stated, kindly stick with the right font color for both the paragraph and the h1